pkg/reconciler/v1alpha1/clusteringress/clusteringress.go

```diff
@@ -242,6 +236,11 @@ func (c *Reconciler) reconcile(ctx context.Context, ci *v1alpha1.ClusterIngress)
 	return nil
 }
 
+func enablesAutoTLS(ctx context.Context) bool {
```

Comment:
nit - this is probably better named enableReconcilingGateway, especially given this isnt 1:1 with tlsMode == Auto.

Reply:
I am gonna keep it as `enableAutoTLS` temporarily as currently I use `autoTLS` to control auto-TLS feature. I will probably change it to `enableReconcilingGateway` when I add `enable-reconciling-gateway` into `config-istio.yaml`.